src/core/request-body-search.js

```diff
@@ -406,6 +406,74 @@ class RequestBodySearch {
         return this;
     }
 
+    /**
+     * Computes a document property dynamically based on the supplied `script`.
```

Comment:
Could you add a note here mentioning that this feature was added to Elasticsearch in v7.11?